Makefile

```diff
@@ -88,7 +88,7 @@ endif
 
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress
-MANPAGES=ctr.1 containerd.8 containerd-config.1 containerd-config.toml.5
+MANPAGES=ctr.8 containerd.8 containerd-config.1 containerd-config.toml.5
```

Comment:
`containerd-config` should be moved as well?

Reply:
good one; I guess it's related to the `containerd` command, so would also be "admin" and/or require root? I can update